adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+from string import ascii_uppercase
+from random import randint , choice
+
+
+
 def draw_letters():
-    pass
+    letter_pool = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+    hand = []
+
+
+    while len(hand) < 10:
+        pick = choice(ascii_uppercase)
```

Comment:
A more descriptive name for `pick` might be `random_letter`. 

Also `ascii_uppercase` evaluates to a string with 26 letters in it. However, we need to select from a pool of 98 letters. Therefore, when we use `choice(ascii_uppercase)` the likelihood of selecting 'A' is 1 out of 26, but it should actually be 9 out of 98. 

How can you use `letter_pool` to build up a list of letters that is 98 letters long according to the distribution that the dictionary outlines? _Please let me know as a response to this comment._

Reply:
I know how to implement it. I chose dictionary mutating because I was thinking about productivity, as iterating a dictionary and creating a list is a more resource-consuming process (in my mind).

`   list_of_letters = []
    for letter in LETTER_POOL:
        list_of_letters.extend(letter * LETTER_POOL[letter])`

Reply:
Do I need to replace all the logic tied to a dictionary `letter_pool` with a "list of letters" logic?